test/deploy/os_ova_install.py

```diff
@@ -18,19 +23,26 @@
 class os_ova_install(fit_common.unittest.TestCase):
     def test01_install_ova_template(self):
         ovafile = fit_common.ARGS_LIST['template']
+        numvms = int(fit_common.ARGS_LIST['numvms'])
         # Check for ovftool
         self.assertEqual(fit_common.subprocess.call('which ovftool', shell=True), 0, "FAILURE: 'ovftool' not installed.")
         # Ping for valid ESXi host
         self.assertEqual(fit_common.subprocess.call('ping -c 1 ' + fit_common.ARGS_LIST['hyper'], shell=True), 0, "FAILURE: ESXi hypervisor not found.")
-        # Shutdown previous ORA
-        if fit_common.subprocess.call('ping -c 1 ' + fit_common.ARGS_LIST['ora'], shell=True) == 0:
-            fit_common.remote_shell('shutdown -h now')
-            fit_common.time.sleep(5)
 
         # Run probe to check for valid OVA file
         rc = fit_common.subprocess.call("ovftool " + ovafile, shell=True)
         self.assertEqual(rc, 0,'Invalid or missing OVA file: ' + ovafile)
 
+        # check for number of virtual machine and stack ID as number
+        self.assertTrue(numvms < 100, "Number of vms should not be greater than 99")
+        if numvms > 1:
+            self.assertTrue(fit_common.ARGS_LIST['stack'].isdigit(), "Stack ID must be a number if numvms is greater than 1")
```

Comment:
Here if numvms==1, then we use MAC address from stack_config file which has stack ID either number or word, And if numvm>1 then we defined MAC address from stack ID which must be number in order to create MAC address